### ISSUE TITLE
Feature Request: Give KP For Key Vanilla Achievements

### ISSUE THREAD

**Issue by bmemike** [Iskallia/Vault-public#246]
Knowledge points are extremely difficult to acquire early on - and a few early in progression would be very handy/appreciated.

Please consider granting a very limited number of KP for significant vanilla progression steps. 

Ex:
1) first dragon kill
2) first wither kill
3) kill 3x elder guardians

And maybe that’s it!

Or possibly unlock specific mods for these achievements instead of granting KP that can be spent on anything. 

Research like Decorator is hard to justify early on while KP are so scarce, but that’s when ppl would get the most use out of a mod like this while doing base building.

Thank you.

**Comment by Aattika** [comment]
I agree with you, i run more than 20 vaults, got omega, and epic treasure rooms (total of 7), and still not a single star core. I got one star core from a boss crate, thats all. (I unlocked looter 2 before the epic/omega runs)

I have knowledge essence, and star shard, but the core is impossible to get. Maybe there could be a recipe for that, or something. Really hard to unlock any mod.

**Comment by Euroam** [comment]
@Aattika there is actually a recipe which is not showing on jei, but it is expensive, the recipe is vault diamond block in the middle surrounded by larimar gem
